@ecomplus/storefront-components/src/scss/TheProduct.scss

```
@@ -130,6 +130,27 @@
     }
   }
 
+  .sale-timer {
```

```suggestion
  &__sale-timer {
```

https://en.bem.info/methodology/css/
Pelo mesmo motivo do ID, sem prefixo o class name pode bater com de outro elemento da página, BEM é uma forma de dar escopo

---

ok